controllers/components/codeflare/codeflare.go

```diff
@@ -7,7 +7,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	componentsv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1"
+	componentsv1alpha1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1alpha1"
```

Comment:
@lburgazzoli I'm wondering while there is no dependency of 2 versions simultaneously in the code, why to put the version in the namespace? It causes such huge renaming like this for example.

Reply:
agree

FWIW, maybe we can use something like `componentsApi` to point to the storage version of the API is there is a need to have an alias. So we would have a stable naming for the actual API and aliases for alternatives if needed.

Reply:
I am okay with updating it to `componentsApi`. I agree keeping names version agnostic is helpful.